### PR TITLE
S03 us03

### PR DESCRIPTION
**Pull request by Kino-ki**

### CODE REVIEW

**Review by Jubdc — APPROVED** [review]

**Review by Halicksse — APPROVED** [review]

File: backend/database/schema.sql [hunk 1]
```diff
@@ -26,28 +26,28 @@ CREATE TABLE
...
     );
 
 CREATE TABLE
     artwork (
         id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
-        path_pic VARCHAR(255) NOT NULL,
+        path_pic VARCHAR(255) NULL,
         title VARCHAR(255) NULL,
-        adress VARCHAR(255) NOT NULL,
+        coordinates POINT NOT NULL,
```
Comment (ioayoub) [inline backend/database/schema.sql:50]: Latitude, longitude fields as we said earlier

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: backend/database/schema.sql (hunk 1 above)
Comment (Kino-ki) [reply backend/database/schema.sql]: done!

File: frontend/src/main.jsx [hunk 2]
```diff
@@ -13,11 +14,16 @@ const router = createBrowserRouter([
       {
         path: "/map",
         element: <MapPage />,
+        loader: () => fetch(`http://localhost:3310/api/artwork/`),
```
Comment (ioayoub) [inline frontend/src/main.jsx:17]: Use your .env variable here

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by Lambseb — APPROVED** [review]

**Review by Lambseb — APPROVED** [review]

File: frontend/src/main.jsx (hunk 2 above)
Comment (Kino-ki) [reply frontend/src/main.jsx]: done !

**Review by Jhanemba64 — APPROVED** [review]

**Review by Lambseb — APPROVED** [review]

**Review by ioayoub — APPROVED** [review]

**Pull request merged**